client/plots/matrix.js

```diff
@@ -159,24 +159,24 @@ export class Matrix {
 			this.dom.loadingDiv.html('').style('display', '').style('position', 'absolute').style('left', '45%')
 
 			// may skip data requests when changes are not expected to affect the request payload
-			if (this.stateDiff.nonsettings) {
-				// reset highlighted dendrogram children to black when data request is triggered
-				delete this.clickedChildren
-
-				const promises = []
-				// get the data
-				if (this.setHierClusterData) promises.push(this.setHierClusterData())
-				promises.push(this.setData())
-				this.dom.loadingDiv.html('Processing data ...')
-				await Promise.all(promises)
-				applyLegendValueFilter(this)
-				if (this.combineData) this.combineData()
-				// tws in the config may be filled-in based on applicable server response data;
-				// these filled-in config, such as tw.term.values|category2samplecount, will need to replace
-				// the corresponding tracked state values in the app store, without causing unnecessary
-				// dispatch notifications, so use app.save()
-				this.app.save({ type: 'plot_edit', id: this.id, config: this.config })
-			}
+			//if (this.stateDiff.nonsettings) {
```

Comment:
@airenzp @xzhou82  Commenting out the condition for the code block will slow down the matrix update, since server request + gene clustering will be triggered even for trivial changes.